hazelcast/src/main/java/com/hazelcast/map/impl/mapstore/writebehind/ArrayWriteBehindQueue.java

```diff
@@ -160,4 +160,26 @@ public boolean isEnabled() {
         return delayedEntries;
     }
 
+    /**
+     * Returns supplied number of entries from the start.
+     *
+     * @param count number of entries to return.
+     * @return list of entries
+     */
+    @Override
+    public List<DelayedEntry> get(int count) {
```

Comment:
I still don't understand why here's a `count` parameter. It's only used for list initialization (`new ArrayList<DelayedEntry>(count)`) but it has nothing to do with `@param count number of entries to return`.